juniper_official/Solutions/Pfe-wedge-detection/eachip-pattern-set.rule

```diff
@@ -0,0 +1,489 @@
+                constant syslog-trigger-color {
+                    value red;
+                }
+                constant syslog-trigger-message {
+                    value "If only a few errors occur, then clear the Alarm.  Else, contact JTAC.";
+                }
+                key-fields fpc_number;
+            }
+            pattern EACHIP_EVENT036 {
+                event-id EACHIP_EVENT036;
+                filter "fpc%{NUMBER:fpc_number}.*MQSS.*FI.*Parity error detected for bank.*of CP table";
+                description "The \"Parity error detected for bank .. of CP table\" message reports a hardware parity error detected in the EACHIP Fabric input block. This is due to a transient hardware SRAM error within EACHIP Fabric input block.";
+                constant syslog-trigger-color {
+                    value red;
+                }
+                constant syslog-trigger-message {
+                    value "If only a few errors occur, then clear the Alarm.  Else, contact JTAC.";
+                }
+                key-fields fpc_number;
+            }
+            pattern EACHIP_EVENT037 {   
+                event-id EACHIP_EVENT037;
+                filter "fpc%{NUMBER:fpc_number}.*HMM.*Lane.*TX RLL event ";
+                description "The \"TX RLL event\" message reports Run Length Limit (RLL) event on a Hybrid Memory Link. This indicates a Run Length Limit event on scrambled links and Junos OS will automatically retry to recover. Successful recovery will be confirmed with the second debug message with State: values between 0x20 - 0x23.​ A response packet with a benign WARNING state is received from HMC - State: 0x20";
+                constant syslog-trigger-color {
+                    value red;
+                }
+                constant syslog-trigger-message {
+                    value "No action if only a few occurrences.  If seen repeatedly, contact JTAC.";
+                }
+                key-fields fpc_number;
+            }
+            pattern EACHIP_EVENT038 {
+                event-id EACHIP_EVENT038;
+                filter "fpc%{NUMBER:fpc_number}.*MQSS.*BCMF CBUF.*Parity error corrected for buffer free list memory for bank";
+                description "The \"Parity error corrected for buffer free list memory for bank\" message reports a transient hardware error which was automatically corrected. This is due to a transient hardware SRAM error with EACHIP.";
+                constant syslog-trigger-color {
+                    value red;
+                }
+                constant syslog-trigger-message {
+                    value "If single log, clear alarm from CLI ('clear chassis fpc errors fpc-slot <fpc> error-code <code>', where <code> is from 'show chassis alarms').  If multiple logs, reset FPC and monitor.";
```

Comment:
And here..

Reply:
@ConstrainedRandomVar , added trigger based on count